benchmark/index.ts

```diff
@@ -67,44 +67,44 @@ const suite = new Benchmark.Suite();
 // Benchmarking
 suite.add('got - promise', {
 	defer: true,
-	fn: async (deferred: {resolve(): void}) => {
+	fn: async (deferred: {resolve: () => void}) => {
```

Comment:
TS team recommends this style as it's slightly stricter.

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/method-signature-style.md